steps/report.sh

```diff
@@ -57,6 +58,67 @@ fi
 
 sort -o "${list}" "${list}"
 
+# Create the aggregation table LaTeX file
+aggregation_table=${TARGET}/temp/aggregation_table.tex
+rm -f "${aggregation_table}"
+touch "${aggregation_table}"
```

Comment:
@dzhovi I believe, these two command may be replaced with one: `echo > "${aggregation_table}"`

Reply:
@yegor256 fixed